composer.json

```diff
@@ -7,8 +7,8 @@
     "license": "MIT",
     "authors": [
         {
-            "name": "Clément Guillemain",
-            "homepage": "http://clementguillemain.fr",
+            "name": "Newerton Vargas de Araujo",
```

Comment:
Don't remove base author. We're going to add a new entry in authors to thank the community with link to : https://github.com/Sybio/ImageWorkshop/graphs/contributors